src/Telegram.php

```diff
@@ -815,14 +816,15 @@ protected function ucfirstUnicode($str, $encoding = 'UTF-8')
     /**
      * Enable Botan.io integration
      *
-     * @param  $token
+     * @param  string $token
+     * @param  integer $timeout
      *
      * @return \Longman\TelegramBot\Telegram
      * @throws \Longman\TelegramBot\Exception\TelegramException
      */
-    public function enableBotan($token)
+    public function enableBotan($token, $timeout = 3)
```

Comment:
I'm just wondering if we might want to have an array here for extra parameters, to prevent issues in the future if new parameters might become available.
Having an array, we just add indexes and previous code still works. Adding extra parameters to the method becomes difficult to manage.

Here an example of the problem I'm trying to explain:
https://github.com/akalongman/php-telegram-bot/pull/346/files#diff-04c6e90faac2675aa89e2176d2eec7d8L245